File: Stravopodis_Safaric/QL/src/uva/ql/ast/ASTNode.java
package uva.ql.ast;

public abstract class ASTNode {
	
	protected CodeLines codeLines;
	
	public ASTNode(){
		super();
	}
	
	public ASTNode(CodeLines _codeLines){
		this.codeLines = _codeLines;
	}
}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/ASTNode.cs


namespace AST.Nodes
{
    public abstract class ASTNode
    {
        private readonly PositionInText position;
        protected ASTNode(PositionInText position)
        {
            this.position = position;
        }
        
        public PositionInText GetPosition()
        {
            return position;
        }
    }
}
